taskcluster/focus_android_taskgraph/loader/__init__.py

```diff
@@ -0,0 +1,73 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import, print_function, unicode_literals
```

Comment:
Since you'll now be running with Python 3, you can update these files here. I like to use the [pyupgrade](https://github.com/asottile/pyupgrade) tool to do it automatically:
```
$ pyupgrade --py36-plus `find taskcluster -name "*.py" -type f`
```

from the root of the repo.

Reply:
hrm, should this part use the backticks? find taskcluster -name "*.py" -type f as shown above? I get command not found when I copy and type that verbatim. Also, what exactly is this doing? I took a quick look in the docs but its not clear to me.

Reply:
Did you install `pyupgrade`? Otherwise I guess you don't have the `find` command or maybe backticks aren't supported in your shell.

Pyupgrade will automatically re-format your code to use features from the minimum specified python version. So the above will e.g, auto-remove future imports, get rid of `six`, convert `.format()` to `f-strings`, etc. Unfortunately `pyupgrade` only supports modifying a single file a a time, so you need to use shell commands to pass it the list of files to re-format (thus the `find` command).

You could try using `pyupgrade --py36-plus $(find taskcluster -name "*.py" -type f)` instead? If that still doesn't work, try getting the `find` command to work on its own first.

Reply:
> You could try using `pyupgrade --py36-plus $(find taskcluster -name "*.py" -type f)` instead? If that still doesn't work, try getting the `find` command to work on its own first.

Aha! That worked.